src/ExpoClient.ts

```diff
@@ -75,13 +77,13 @@ export class Expo {
    * sized chunks.
    */
   async sendPushNotificationsAsync(messages: ExpoPushMessage[]): Promise<ExpoPushTicket[]> {
+    const useFCM = this.useFCM ? '?useFcmV1=true' : '';
```

Comment:
Could you use the `URL` API instead? With even more string concatenation, the risk of strings being treated as URL components goes up.

Also, it would be good to have the logic be: if the option is set to true or false, set the query parameter, otherwise omit the query parameter and let the server choose its default behavior. That way people can explicitly opt out of using FCM v1 too.

Reply:
Makes sense, will sort tomorrow.

Reply:
Just realised our usernames are an anagram 😄